src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -407,14 +414,13 @@ private static EnvironmentStatus WaitForReleaseEnvironmentCompletion(string proj
                 {
                     cancellationToken.ThrowIfCancellationRequested();
 
-                    Task.Delay(5 * 1000, cancellationToken).Wait();
+                    Task.Delay(5000, cancellationToken).Wait();
 
                     environmentStatus = GetReleaseEnvironment(projectName, release.Id, environmentId, releaseClient, cancellationToken).Status;
 
-                    if(!(environmentStatus.Equals(EnvironmentStatus.InProgress)
-                          || environmentStatus.Equals(EnvironmentStatus.Queued)
-                          || environmentStatus.Equals(EnvironmentStatus.NotStarted)
-                          || environmentStatus.Equals(EnvironmentStatus.Scheduled)))
+                    if (environmentStatus != EnvironmentStatus.InProgress
```

Comment:
this change keeps the tool from spinning for 10 minutes waiting for a release that isn't actually happening.

Reply:
Do we need any of this release stuff at all?  What is the release for?

Reply:
The comments make it seem like it is supposed to do symbol publishing for the build we're inserting. But it's not clear to me if that is still something that RIT users need to be worried about, or whether it's something that just happens as part of the signed build pipeline. Can you shed any light @brettfo?